dotnet/samples/Demos/VectorStoreRAG/DataLoader.cs

```diff
@@ -21,14 +23,14 @@ internal sealed class DataLoader<TKey>(
     ITextEmbeddingGenerationService textEmbeddingGenerationService) : IDataLoader where TKey : notnull
 {
     /// <inheritdoc/>
-    public async Task LoadPdf(string pdfPath, CancellationToken cancellationToken)
+    public async Task LoadPdf(string pdfPath, int maxDegreeOfParallelism, int betweenBatchDelayInMs, CancellationToken cancellationToken)
```

Comment:
nit: instead of `maxDegreeOfParallelism`, should we call it simply `batchSize`?

Reply:
Sure, done.